certified-connectors/ZohoSign/readme.md

```diff
@@ -0,0 +1,56 @@
+﻿
+**Zoho-Sign**
+
+Zoho Sign lets you digitally sign and verify legally binding documents without the hassles of transporting paper documents. It's a complete digital signature app for business signatories.
+
```

Comment:
@ZohoSign-integ Please refer the templates provided for readme.md file and add required sections and format the document.